migrator/src/main/java/uk/nhs/digital/ps/migrator/config/ExecutionParameters.java

```diff
@@ -103,6 +104,14 @@ public void setTaxonomyMappingImportPath(Path taxonomyMappingImportPath) {
         this.taxonomyMappingImportPath = taxonomyMappingImportPath;
     }
 
+    public Path getFieldMappingImportPath() {
+        return fieldMappingImportPath;
```

Comment:
Could be worth emphasising that this is CI-specific mapping. Other properties use prefix  'nesstar' for this purpose, so how about renaming this property and the command line arg to, say, `nesstarFieldMappingImportPath`?

Both for consistency and to indicate that this should only be used for this specific type of mapping file (so as to not to suggest that this is for some generic/universal type of mapping).